go-app-ussd_registration.js

```diff
@@ -578,11 +580,24 @@ go.app = (function() {
       return new FreeText(name, {
         question: get_content(name).context(),
         check: function(content) {
+          var givendate = content;
```

Comment:
This variable looks unnecessary, you could just rename content to givendate above.

Reply:
Alright